src/mixins/filing-template-mixin.ts

```diff
@@ -82,46 +86,44 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
   @Action setCustodianOfRecords!: ActionBindingIF
 
   /**
-   * Constructs a filing body from store data. Used when saving a filing.
+   * Builds an incorporation filing from store data. Used when saving a filing.
    * @returns the filing body to save
    */
   buildIncorporationFiling (): IncorporationFilingIF {
-    // Build filing.
+    // Build the main filing.
     const filing: IncorporationFilingIF = {
-      filing: {
```

Comment:
if the interface now applied at a different point when parsing a filing? 
I guess it is confusing, as the only time you'd expect to see the `filing` wrapper is in a GET on filings or tasks response, so this probably makes sense in this context!

Reply:
There is no change to the parsing, only to the data object we send to the API.

Reply:
Oh, so we ARE doing away with the wrapper now, or is it applied at the point of request? 
Seems like a change that would affect more than just this filing if there was changes on the api side.

Reply:
oh scratch that, i see now that it's double wrapped. 🤦